spec/models/event_spec.rb

```diff
@@ -41,4 +41,31 @@
       it { is_expected.to_not be_valid }
     end
   end
+
+  describe '.requiring_notification' do
+    subject(:events_requiring_notification) { Event.requiring_notification }
+    context 'when there are events aligning with the NOTIFICATION_PERIOD' do
+      let(:required_notif) do
+        FactoryGirl.create(
+          :event, date: Time.zone.today - Event::NOTIFICATION_PERIOD
+        )
+      end
+      let(:unrequired_notif) do
+        FactoryGirl.create(
+          :event, date: Time.zone.today + Event::NOTIFICATION_PERIOD
+        )
+      end
+      it 'returns an array of Events requiring notification' do
+        expect(events_requiring_notification).to include(required_notif)
+      end
+      it 'doesnt include Events that arent requiring notification' do
```

Comment:
Double negative - reads funny.
it 'only includes events that require notification' or something? minor, and doesn't really matter.